flixel/addons/util/position/FlxDynamicPosition.hx

```diff
@@ -0,0 +1,215 @@
+package flixel.addons.util.position;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.math.FlxPoint;
+
+using flixel.addons.util.position.FlxDynamicPosition;
+
+/**
+ * @author DleanJeans
+ * @link https://twitter.com/DleanJeans
+ * 
+ * An util class for positioning Dynamic objects through static extension
+ * For FlxObjects, use FlxPosition
+ * 
+ * Import:
+ * 
+ * using flixel.addons.util.position.FlxDynamicPosition;
+ * 
+ * To import both FlxPosition and FlxDynamicPosition,
+ * import FlxDynamicPosition first and then FlxPosition:
+ * 
+ * using flixel.addons.util.position.FlxDynamicPosition;
+ * using flixel.addons.util.position.FlxPosition;
+ */
+
+class FlxDynamicPosition {
+	public static var screenCenter(get, null):FlxPoint;
+	public static var screenTopLeft(get, null):FlxPoint;
+	public static var screenMidTop(get, null):FlxPoint;
+	public static var screenTopRight(get, null):FlxPoint;
+	public static var screenMidRight(get, null):FlxPoint;
+	public static var screenBottomRight(get, null):FlxPoint;
+	public static var screenMidBottom(get, null):FlxPoint;
+	public static var screenBottomLeft(get, null):FlxPoint;
+	public static var screenMidLeft(get, null):FlxPoint;
+	
+	public static inline function getTopLeft(object:Dynamic):FlxPoint {

```

Comment:
I don't think we should promote the use of `Dynamic` in this way, it's bad practice and bad for performance.